api/src/utils/get-date-formatted.test.ts

```diff
@@ -0,0 +1,36 @@
+import { afterEach, beforeEach, expect, test, vi } from 'vitest';
+
+import { getDateFormatted } from './get-date-formatted';
+
+beforeEach(() => {
+	vi.useFakeTimers();
+});
+
+afterEach(() => {
+	vi.useRealTimers();
+});
+
+function getUtcDateForString(date: string) {
+	const now = new Date(date);
+
+	// account for timezone difference depending on the machine where this test is ran
+	const timezoneOffsetInMinutes = now.getTimezoneOffset();
+	const timezoneOffsetInMilliseconds = timezoneOffsetInMinutes * 60 * 1000;
+	const nowUTC = new Date(now.valueOf() + timezoneOffsetInMilliseconds);
+
+	return nowUTC;
+}
+
+// currently only uses January and November dates to account for daylight saving
+test.each([
+	{ utc: '2023-01-01T01:23:45.678Z', expected: '20230101-12345' },
+	{ utc: '2023-01-11T01:23:45.678Z', expected: '20230111-12345' },
+	{ utc: '2023-11-01T01:23:45.678Z', expected: '20231101-12345' },
+	{ utc: '2023-11-11T12:34:56.789Z', expected: '20231111-123456' },
```

Comment:
```suggestion
	{ utc: '2023-11-01T01:23:45.678Z', expected: '20231101-12345' },
	{ utc: '2023-11-11T12:34:56.789Z', expected: '20231111-123456' },
	{ utc: '2023-06-01T01:23:45.678Z', expected: '20230601-12345' },
	{ utc: '2023-06-11T12:34:56.789Z', expected: '20230611-123456' },
```
Just to be on the safe side testing a summer-time (mar-nov) date would be good to include